### ISSUE TITLE
Pre-work Checklist: Developer: Phillip Sanchez

### ISSUE THREAD

**Issue by plang-psm** [hackforla/website#3511]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events.

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Special Notes
1. It may take you a few weeks to finish this issue, because part of it is learning how to provide updates on issues that take more than a week to complete.    Please keep this issue open until you have been able to do all the steps.  
2. Usually we don't want to you have more than one issue assigned to you at a time, this issue is the exception, because it is instructions on how to work on other issues.
3. The action items listed below should mostly be worked on in a sequential order. However, you don't have to wait on one if you can proceed with the others, For instance, you don't have to wait for attending a weekly meeting before setting up your dev environment.  

### Action Items

- [X] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [X] Share your GitHub handle and Gmail address with a merge team member or technical lead so you can be added to the Google Drive
  - [X] To find contact information for the merge team members and technical leads, please take a look at our [Meet the Team wiki page](https://github.com/hackforla/website/wiki/Meet-the-Team)
- [X] Also, confirm with a merge team member or a technical lead that they have added you to the Google Calendar invites for our Zoom meetings
- [X] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [X] After you have finished adding yourself to the roster, let a merge team member or a technical lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [X] Once added to the website-write team:
  - [X] Self Assign this issue (gear in right side panel)
  - [X] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label (gear in right side panel)
  - [X] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [X] Signup for a [Figma](https://Figma.com) account
- [X] Attend weekly team meetings:
  - [X] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [X] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [X] Note: There are no meetings on the 1st-7th of every month.
- [X] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [X] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [X] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [X] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [X] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [X] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [X] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [X] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment. (Note: Please provide estimates on all the issues that you pick up, going forward.)
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [X] Time spent so far: Copy the question below into a new comment below (in this pre-work issue only) and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [X] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue. **This update should be done every week for every issue that you are assigned to**. The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [X] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [ X Additional notes:
   - [X] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [X] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [X] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [X] Read the [Start Here - Developers](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=8583%3A0) in Figma
- [X] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points.
- Then, add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question. In addition, you will still need to post a Slack message or bring it up in meeting so we know you need help; see below for how to do that.
- Also, you can post your question on the hfla-site slack channel and link the issue you're working on, so other developers can see and respond.
- Lastly, you can add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting. Please bring it during the meeting that you need help.

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by plang-psm** [comment]
Availability for this week: 3 hrs
   My estimated ETA for completing this issue: End of 9/2/22

    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
6hours

    Provide Update
    1. Progress - Awaiting review and merge
    2. Blockers - none
    3. Availability - 3 hrs
    4. ETA - End of 9/2/22

**Comment by plang-psm** [comment]
All items have been completed.

**Comment by ExperimentsInHonesty** [comment]
@plang-psm I noticed that you have taken on another medium size issue instead of a large size issue.  Just checking if that was intentional. 

I am reopening this issue so that we can see you do an update on a coding issue (although I appreciate that you put an update on this issue.

**Comment by plang-psm** [comment]
@ExperimentsInHonesty Hi Bonnie, my apologies for closing out. I must've misread the issues section. I'll close out once I complete a large issue. And yes, I did pick up another medium issue because my previous one was significantly reduced due to some complications with the issue being outdated This resulted in me only having to do a simple link change. I didn't feel like I got to experience a real medium issue and decided to assign myself to another before moving onto a large issue.

**Comment by plang-psm** [comment]
Progress: My current progress is awaiting review for my medium issue. The only thing left to complete is a large issue.

Blockers: No blockers at this time other than waiting for review but I did encounter many blockers while trying to do a medium issue that were out of my control.

Availability: 4hrs

ETA: TBD I plan to take my first large issue within the next week when my medium issue is merged.

Pictures (optional): N/A.

**Comment by plang-psm** [comment]
Progress: Resubmitted my medium issues, and need to complete a large issues.
Blockers: None
Availability: 2hrs
ETA: Medium issue completed, Large issue TBD
Pictures (optional): N/A

**Comment by plang-psm** [comment]
Progress: Resubmitted my medium issues, awaiting review. Will jump into Large issue while I await review.
Blockers: None
Availability: 8hrs
ETA: TBD
Pictures (optional): N/A

**Comment by plang-psm** [comment]
Complete

**Comment by HackforLABot** [comment]
## Activity Log: plang-psm
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- plang-psm opened: https://github.com/hackforla/website/issues/3511 at 08/30/2022, 08:09 PM PDT
- plang-psm assigned: https://github.com/hackforla/website/issues/3511 at 08/30/2022, 08:47 PM PDT
- plang-psm assigned: https://github.com/hackforla/website/issues/2694#issuecomment-1099560647 at 08/30/2022, 09:08 PM PDT
- plang-psm unassigned: https://github.com/hackforla/website/issues/2694#issuecomment-1232437217 at 08/30/2022, 09:22 PM PDT
- plang-psm assigned: https://github.com/hackforla/website/issues/2831#issuecomment-1047131306 at 08/30/2022, 09:24 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/issues/2831#issuecomment-1232446729 at 08/30/2022, 09:26 PM PDT
- plang-psm opened: https://github.com/hackforla/website/pull/3515 at 09/01/2022, 04:43 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/issues/3511#issuecomment-1234930621 at 09/01/2022, 05:11 PM PDT
- plang-psm PR was merged: https://github.com/hackforla/website/pull/3515#event-7317225546 at 09/03/2022, 08:27 AM PDT
- plang-psm issue closed by PR 3515: https://github.com/hackforla/website/issues/2831#event-7317225581 at 09/03/2022, 08:27 AM PDT
- plang-psm assigned: https://github.com/hackforla/website/issues/3097#issuecomment-1116785375 at 09/04/2022, 06:21 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/issues/3097#issuecomment-1236461319 at 09/04/2022, 06:23 PM PDT
- plang-psm opened: https://github.com/hackforla/website/pull/3519 at 09/04/2022, 06:45 PM PDT
- plang-psm PR was merged: https://github.com/hackforla/website/pull/3519#event-7326375702 at 09/05/2022, 07:22 PM PDT
- plang-psm issue closed by PR 3519: https://github.com/hackforla/website/issues/3097#event-7326375749 at 09/05/2022, 07:22 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/pull/3519#issuecomment-1238733867 at 09/06/2022, 04:02 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/pull/3519#issuecomment-1238735678 at 09/06/2022, 04:05 PM PDT
- plang-psm assigned: https://github.com/hackforla/website/issues/3520#issuecomment-1237381656 at 09/06/2022, 04:59 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/issues/3520#issuecomment-1238766999 at 09/06/2022, 05:08 PM PDT
- plang-psm opened: https://github.com/hackforla/website/pull/3521 at 09/06/2022, 07:46 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/pull/3521#issuecomment-1241225802 at 09/08/2022, 01:57 PM PDT
- plang-psm PR was merged: https://github.com/hackforla/website/pull/3521#event-7352440576 at 09/08/2022, 06:05 PM PDT
- plang-psm issue closed by PR 3521: https://github.com/hackforla/website/issues/3520#event-7352440619 at 09/08/2022, 06:05 PM PDT
- plang-psm assigned: https://github.com/hackforla/website/issues/1750#issuecomment-1163664110 at 09/15/2022, 05:54 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/issues/1750#issuecomment-1248797467 at 09/15/2022, 05:55 PM PDT
- plang-psm opened: https://github.com/hackforla/website/pull/3558 at 09/15/2022, 09:48 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/pull/3557#issuecomment-1248918466 at 09/15/2022, 09:55 PM PDT
- plang-psm submitted review: https://github.com/hackforla/website/pull/3557#pullrequestreview-1110117397 at 09/15/2022, 10:42 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/issues/3511#issuecomment-1249958390 at 09/16/2022, 05:29 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/pull/3558#issuecomment-1250162288 at 09/17/2022, 05:27 PM PDT
- plang-psm opened: https://github.com/hackforla/website/pull/3564 at 09/19/2022, 01:15 PM PDT
- plang-psm closed: https://github.com/hackforla/website/pull/3558#event-7416718374 at 09/19/2022, 01:19 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/pull/3558#issuecomment-1251508451 at 09/19/2022, 01:20 PM PDT
- plang-psm closed as completed: https://github.com/hackforla/website/issues/3511#event-7426987232 at 09/20/2022, 07:57 PM PDT
- plang-psm PR was merged: https://github.com/hackforla/website/pull/3564#event-7449346733 at 09/23/2022, 10:52 AM PDT
- plang-psm issue closed by PR 3564: https://github.com/hackforla/website/issues/1750#event-7449346835 at 09/23/2022, 10:52 AM PDT
- plang-psm assigned: https://github.com/hackforla/website/issues/2021#issuecomment-920303375 at 09/29/2022, 07:07 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/issues/2021#issuecomment-1263020286 at 09/29/2022, 07:08 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/pull/3586#issuecomment-1263034556 at 09/29/2022, 07:36 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/pull/3590#issuecomment-1263034717 at 09/29/2022, 07:36 PM PDT
- plang-psm submitted review: https://github.com/hackforla/website/pull/3590#pullrequestreview-1126196828 at 09/29/2022, 08:06 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/issues/3511#issuecomment-1264705918 at 10/02/2022, 11:32 AM PDT
- plang-psm unassigned: https://github.com/hackforla/website/issues/2021#issuecomment-1263020286 at 10/02/2022, 11:43 AM PDT
- plang-psm assigned: https://github.com/hackforla/website/issues/2210#issuecomment-1255890741 at 10/06/2022, 01:04 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/issues/2210#issuecomment-1270620171 at 10/06/2022, 01:07 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/issues/2210#issuecomment-1270635660 at 10/06/2022, 01:23 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/pull/3599#issuecomment-1270653032 at 10/06/2022, 01:40 PM PDT
- plang-psm submitted review: https://github.com/hackforla/website/pull/3599#pullrequestreview-1133624080 at 10/06/2022, 01:51 PM PDT
- plang-psm submitted review: https://github.com/hackforla/website/pull/3599#pullrequestreview-1133624989 at 10/06/2022, 01:52 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/pull/3597#issuecomment-1270686250 at 10/06/2022, 01:55 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/pull/3597#issuecomment-1272602670 at 10/09/2022, 11:32 AM PDT
- plang-psm submitted review: https://github.com/hackforla/website/pull/3597#pullrequestreview-1138564400 at 10/12/2022, 12:21 AM PDT
- plang-psm commented on: https://github.com/hackforla/website/issues/2210#issuecomment-1278500180 at 10/13/2022, 10:24 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/issues/3511#issuecomment-1279152657 at 10/14/2022, 08:25 AM PDT
- plang-psm submitted review: https://github.com/hackforla/website/pull/3597#pullrequestreview-1144804249 at 10/17/2022, 01:59 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/pull/3643#issuecomment-1285986923 at 10/20/2022, 11:43 AM PDT
- plang-psm submitted review: https://github.com/hackforla/website/pull/3643#pullrequestreview-1149797296 at 10/20/2022, 11:45 AM PDT
- plang-psm commented on: https://github.com/hackforla/website/issues/2210#issuecomment-1299304102 at 11/01/2022, 03:22 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/issues/2210#issuecomment-1299347037 at 11/01/2022, 04:19 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/issues/2210#issuecomment-1299354106 at 11/01/2022, 04:29 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/issues/3511#issuecomment-1299355237 at 11/01/2022, 04:31 PM PDT
- plang-psm commented on: https://github.com/hackforla/website/pull/3702#issuecomment-1308131406 at 11/08/2022, 06:53 PM PST
- plang-psm commented on: https://github.com/hackforla/website/pull/3716#issuecomment-1312612831 at 11/12/2022, 05:35 PM PST
- plang-psm submitted review: https://github.com/hackforla/website/pull/3716#pullrequestreview-1178178756 at 11/12/2022, 05:41 PM PST
- plang-psm commented on: https://github.com/hackforla/website/pull/3721#issuecomment-1316072587 at 11/15/2022, 04:20 PM PST
- plang-psm commented on: https://github.com/hackforla/website/pull/3719#issuecomment-1316073087 at 11/15/2022, 04:21 PM PST
- plang-psm submitted review: https://github.com/hackforla/website/pull/3721#pullrequestreview-1181753401 at 11/15/2022, 05:17 PM PST
- plang-psm submitted review: https://github.com/hackforla/website/pull/3719#pullrequestreview-1181772095 at 11/15/2022, 05:49 PM PST
- plang-psm commented on: https://github.com/hackforla/website/pull/3726#issuecomment-1319304229 at 11/17/2022, 02:43 PM PST
- plang-psm assigned: https://github.com/hackforla/website/issues/3704#issuecomment-1328333526 at 11/27/2022, 12:30 PM PST
- plang-psm commented on: https://github.com/hackforla/website/issues/3511#issuecomment-1328337458 at 11/27/2022, 12:53 PM PST
- plang-psm commented on: https://github.com/hackforla/website/issues/3254#issuecomment-1338485243 at 12/05/2022, 04:29 PM PST
- plang-psm closed as completed: https://github.com/hackforla/website/issues/3254#event-7960554244 at 12/05/2022, 04:29 PM PST
- plang-psm commented on: https://github.com/hackforla/website/issues/2464#issuecomment-1338648923 at 12/05/2022, 06:40 PM PST
- plang-psm commented on: https://github.com/hackforla/website/issues/3511#issuecomment-1397385007 at 01/19/2023, 09:54 AM PST
- plang-psm closed as completed: https://github.com/hackforla/website/issues/3511#event-8314691019 at 01/19/2023, 09:54 AM PST
- plang-psm opened: https://github.com/hackforla/website/issues/3848 at 01/21/2023, 02:57 PM PST
- plang-psm commented on: https://github.com/hackforla/website/pull/3850#issuecomment-1402115633 at 01/24/2023, 07:17 AM PST
- plang-psm merged: https://github.com/hackforla/website/pull/3850#event-8345380637 at 01/24/2023, 07:17 AM PST
- plang-psm reopened: https://github.com/hackforla/website/pull/3850#event-8345380637 at 01/24/2023, 07:18 AM PST
- plang-psm merged: https://github.com/hackforla/website/pull/3850#event-8345388596 at 01/24/2023, 07:18 AM PST
- plang-psm closed by PR 3850: https://github.com/hackforla/website/issues/3396#event-8345388931 at 01/24/2023, 07:18 AM PST
- plang-psm opened: https://github.com/hackforla/website/issues/3882 at 01/26/2023, 07:16 PM PST
- plang-psm merged: https://github.com/hackforla/website/pull/3930#event-8451617413 at 02/06/2023, 12:40 PM PST
- plang-psm closed by PR 3930: https://github.com/hackforla/website/issues/3840#event-8451617741 at 02/06/2023, 12:40 PM PST
- plang-psm opened: https://github.com/hackforla/website/issues/3948 at 02/07/2023, 10:41 AM PST
- plang-psm opened: https://github.com/hackforla/website/issues/3950 at 02/07/2023, 02:11 PM PST
- plang-psm opened: https://github.com/hackforla/website/issues/3951 at 02/07/2023, 02:19 PM PST
- plang-psm opened: https://github.com/hackforla/website/issues/3952 at 02/07/2023, 02:35 PM PST
- plang-psm opened: https://github.com/hackforla/website/issues/3953 at 02/07/2023, 02:48 PM PST
- plang-psm commented on: https://github.com/hackforla/website/issues/3175#issuecomment-1428916537 at 02/13/2023, 04:30 PM PST
- plang-psm commented on: https://github.com/hackforla/website/issues/3175#issuecomment-1429066289 at 02/13/2023, 07:40 PM PST
- plang-psm commented on: https://github.com/hackforla/website/issues/3848#issuecomment-1432235972 at 02/15/2023, 03:42 PM PST
- plang-psm opened: https://github.com/hackforla/website/issues/3986 at 02/15/2023, 03:59 PM PST
- plang-psm opened: https://github.com/hackforla/website/issues/3987 at 02/15/2023, 04:09 PM PST
- plang-psm opened: https://github.com/hackforla/website/issues/3997 at 02/17/2023, 09:54 AM PST
- plang-psm opened: https://github.com/hackforla/website/issues/3998 at 02/17/2023, 10:00 AM PST
- plang-psm opened: https://github.com/hackforla/website/issues/3999 at 02/17/2023, 10:06 AM PST
- plang-psm opened: https://github.com/hackforla/website/issues/4000 at 02/17/2023, 10:32 AM PST
- plang-psm assigned: https://github.com/hackforla/website/issues/3959 at 02/17/2023, 05:16 PM PST
- plang-psm commented on: https://github.com/hackforla/website/issues/3959#issuecomment-1435438501 at 02/17/2023, 05:18 PM PST
- plang-psm opened: https://github.com/hackforla/website/issues/4006 at 02/18/2023, 09:05 PM PST
- plang-psm opened: https://github.com/hackforla/website/issues/4007 at 02/18/2023, 09:08 PM PST
- plang-psm opened: https://github.com/hackforla/website/issues/4008 at 02/18/2023, 09:13 PM PST
- plang-psm opened: https://github.com/hackforla/website/issues/4012 at 02/19/2023, 08:33 PM PST
- plang-psm opened: https://github.com/hackforla/website/issues/4013 at 02/19/2023, 08:37 PM PST
- plang-psm opened: https://github.com/hackforla/website/issues/4014 at 02/20/2023, 07:27 AM PST
- plang-psm opened: https://github.com/hackforla/website/issues/4015 at 02/20/2023, 07:40 AM PST
- plang-psm commented on: https://github.com/hackforla/website/issues/3848#issuecomment-1439311227 at 02/21/2023, 05:30 PM PST
- plang-psm closed as completed: https://github.com/hackforla/website/issues/3848#event-8576290487 at 02/21/2023, 05:30 PM PST
- plang-psm commented on: https://github.com/hackforla/website/pull/3962#issuecomment-1439424868 at 02/21/2023, 08:32 PM PST
- plang-psm commented on: https://github.com/hackforla/website/issues/3274#issuecomment-1441256838 at 02/22/2023, 10:11 PM PST
- plang-psm closed as completed: https://github.com/hackforla/website/issues/3274#event-8588122186 at 02/22/2023, 10:11 PM PST
- plang-psm opened: https://github.com/hackforla/website/issues/4028 at 02/22/2023, 10:49 PM PST
- plang-psm opened: https://github.com/hackforla/website/issues/4045 at 02/25/2023, 11:10 AM PST
- plang-psm opened: https://github.com/hackforla/website/issues/4046 at 02/25/2023, 11:14 AM PST
- plang-psm opened: https://github.com/hackforla/website/issues/4047 at 02/25/2023, 11:18 AM PST
- plang-psm commented on: https://github.com/hackforla/website/issues/4012#issuecomment-1445221249 at 02/25/2023, 02:48 PM PST
- plang-psm commented on: https://github.com/hackforla/website/issues/3959#issuecomment-1445715829 at 02/26/2023, 09:09 PM PST
- plang-psm unassigned: https://github.com/hackforla/website/issues/3959#issuecomment-1445715829 at 02/27/2023, 09:56 PM PST
- plang-psm submitted review: https://github.com/hackforla/website/pull/4107#pullrequestreview-1329367123 at 03/07/2023, 11:30 AM PST
- plang-psm merged: https://github.com/hackforla/website/pull/4107#event-8689636439 at 03/07/2023, 11:30 AM PST
- plang-psm closed by PR 4107: https://github.com/hackforla/website/issues/3925#event-8689636706 at 03/07/2023, 11:30 AM PST
- plang-psm submitted review: https://github.com/hackforla/website/pull/4147#pullrequestreview-1335942893 at 03/11/2023, 02:56 PM PST
- plang-psm merged: https://github.com/hackforla/website/pull/4147#event-8724766717 at 03/11/2023, 02:56 PM PST
- plang-psm closed by PR 4147: https://github.com/hackforla/website/issues/4132#event-8724766741 at 03/11/2023, 02:56 PM PST
- plang-psm commented on: https://github.com/hackforla/website/pull/4144#issuecomment-1465044989 at 03/11/2023, 03:00 PM PST
- plang-psm submitted review: https://github.com/hackforla/website/pull/4144#pullrequestreview-1335943858 at 03/11/2023, 03:11 PM PST
- plang-psm submitted review: https://github.com/hackforla/website/pull/4295#pullrequestreview-1361982290 at 03/28/2023, 03:14 PM PDT
- plang-psm merged: https://github.com/hackforla/website/pull/4295#event-8870068608 at 03/28/2023, 03:14 PM PDT
- plang-psm closed by PR 4295: https://github.com/hackforla/website/issues/4286#event-8870068784 at 03/28/2023, 03:14 PM PDT
- plang-psm submitted review: https://github.com/hackforla/website/pull/4292#pullrequestreview-1361989389 at 03/28/2023, 03:20 PM PDT
- plang-psm merged: https://github.com/hackforla/website/pull/4292#event-8870107679 at 03/28/2023, 03:21 PM PDT
- plang-psm closed by PR 4292: https://github.com/hackforla/website/issues/4287#event-8870107831 at 03/28/2023, 03:21 PM PDT
- plang-psm submitted review: https://github.com/hackforla/website/pull/4290#pullrequestreview-1362016526 at 03/28/2023, 03:51 PM PDT
- plang-psm merged: https://github.com/hackforla/website/pull/4290#event-8870290067 at 03/28/2023, 03:51 PM PDT
- plang-psm closed by PR 4290: https://github.com/hackforla/website/issues/4268#event-8870290202 at 03/28/2023, 03:51 PM PDT
